opm/input/eclipse/EclipseState/Grid/Keywords.hpp

```diff
@@ -34,16 +34,19 @@ struct keyword_info {
     bool multiplier = false;
     bool top = false;
     bool global = false;
+    // it is updated in a later stage when parsing relevant keywords
+    mutable std::size_t num_value = 1;
```

Comment:
I hope to remove the mutable while it appears not easy because it is used a static const variable, 
```c++
static const std::unordered_map<std::string, keyword_info<double>> composition_keywords = {{"XMF", keyword_info<double>{}},
                                                                                           {"YMF", keyword_info<double>{}}
                                                                                          };
```